samples/IISSample/Startup.cs

```diff
@@ -12,6 +13,7 @@ public class Startup
     {
         public void ConfigureServices(IServiceCollection services)
         {
+            services.AddAuthenticationCore();
```

Comment:
This can be removed from samples and tests now that it's added by the main component.